src/qsort.rs

```diff
@@ -0,0 +1,323 @@
+//! Rust implementation of C library function `qsort`
+//!
+//! This code snippet is copied from relibc.
+//! MIT license COPYRIGHT (c) 2018 Redox OS
+
+use crate::{CChar, CInt, CSizeT, CVoid};
+
+/// Rust implementation of C library function `qsort`
```

Comment:
We may need to expand the comments here to explain what `compar` should do?

Reply:
I guess it's defined by the C standard